### PR TITLE
Exercise 4

### PR DESCRIPTION
**Pull request by jswoodburn**

### CODE REVIEW

File: ansible/.env.j2
Comment (JackMeadDev) [inline ansible/.env.j2:6]: If this was real we'd probably want to extract this into a variable (even if it's not secret) as well, though setting this as a default on the variable so we don't always have to type it would definitely be sensible

File: ansible/todo_playbook.yml
Comment (JackMeadDev) [inline ansible/todo_playbook.yml]: Not major, but it's worth avoiding getting into the habit of making permissions '0777' by default - everything will work but security might hate you! This file for example will have secrets in, so avoiding any "other" users from having access at all feels sensible

**Review by JackMeadDev — APPROVED** [review]
Nice work, I can see the app happily running on your node (<http://18.132.191.234:5000/>) and your playbook looks clean and effective 👍 I've left an optional thought or two below, but happy for you to merge this when you're ready

By default, I'll tear down your VMs tomorrow since we won't use them again going forward, but let me know if you'd prefer me to hold off for any reason!

File: ansible/.env.j2
Comment (jswoodburn) [reply ansible/.env.j2:6]: For sure - I think the exercise explicitly said not to extract this just for ease, otherwise I would have done :)

**Pull request closed without merging**
